src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.model;
+
+import java.util.NoSuchElementException;
+
+public class FruitTransaction {
```

Comment:
According to the checklist item 'Don't begin class or method implementation with empty line.', remove the empty line before the class declaration.